templates/index.jinja2

```diff
@@ -1,19 +1,19 @@
+{% extends "base.jinja2" %}
+
 <!--This is the base page. I want to take the links to the data off of this page and create a data tab that will
 bring people to a webpage that looks like this one currently does. I'm thinking of taking the other tabs off and
 having them all exist under browse. Then, I'd put tabs for other parts of the project, such as the simulations. I
 don't know how to make a new page or change the way these links are set up, though-->
 <!DOCTYPE html>
```

Comment:
this doctype does not need to be here.

Reply:
Yeah — that template was full of extra stuff covered by the base template (missed that tag while cleaning it up). I think what goes here and what doesn't need to be will become clearer after the lab session next week—I'll talk a bit about templating in addition to working with static files.